mmdet3d/datasets/lyft_dataset.py

```diff
@@ -415,7 +415,7 @@ def show(self, results, out_dir):
             points = Coord3DMode.convert_point(points, Coord3DMode.LIDAR,
                                                Coord3DMode.DEPTH)
             inds = result['pts_bbox']['scores_3d'] > 0.1
-            gt_bboxes = self.get_ann_info(i)['gt_bboxes_3d'].tensor
+            gt_bboxes = self.get_ann_info(i)['gt_bboxes_3d'].tensor.numpy()
```

Comment:
is `numpy()` here necessary?

Reply:
Yes, not necessary because there are input check in the draw function afterwards, which converts `torch.Tensor` to `np.ndarray`. But I think I can convert it here because I know the input will be drawn in the `np.ndarray` type, and the input pred_bboxes is already `np.ndarray`. It will be strange if we input pred/gt_bbox in different types into one draw function I think.